ironfish-rust-nodejs/src/frost.rs

```diff
@@ -237,3 +237,30 @@ pub fn split_secret(
         public_key_package: bytes_to_hex(&public_key_package_vec),
     })
 }
+
+#[napi(js_name = "PublicKeyPackage")]
+pub struct NativePublicKeyPackage {
+    public_key_package: PublicKeyPackage,
+}
+
+#[napi]
+impl NativePublicKeyPackage {
+    #[napi(constructor)]
+    pub fn new(value: String) -> Result<NativePublicKeyPackage> {
+        let bytes = hex_to_vec_bytes(&value).map_err(to_napi_err)?;
+
+        let public_key_package = PublicKeyPackage::read(&bytes[..]).map_err(to_napi_err)?;
+
+        Ok(NativePublicKeyPackage { public_key_package })
+    }
+
+    #[napi]
+    pub fn identities(&self) -> Result<Vec<Buffer>> {
+        Ok(self
```

Comment:
Because this is now infallible, `Result<Vec<Buffer>>` can be changed to `Vec<Buffer>`, and the `Ok` can be dropped